script/glam/export_csv

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# To be run in glam via:
+#    docker-compose run server ./manage.py import_glean_aggs
+
+set -e
+
+src_project=${SRC_PROJECT:-glam-fenix-dev}
+# TODO: glam-dev-bespoke-nonprod-dataops-mozgcp-net
+dst_project=${DST_PROJECT:-$src_project}
+dataset=${DATASET:-glam_etl_dev}
+product=${PRODUCT:-fenix}   # TODO: set default to org_mozilla_fenix
```

Comment:
I'm going to leave this for another PR, where the prefix will change from `fenix` to `org_mozilla_fenix`.